mitmproxy/filt.py

```diff
@@ -193,12 +193,22 @@ class FBod(_Rex):
     help = "Body"
 
     def __call__(self, f):
-        if f.request and f.request.content:
-            if self.re.search(f.request.get_decoded_content()):
-                return True
-        if f.response and f.response.content:
-            if self.re.search(f.response.get_decoded_content()):
-                return True
+
+        # HTTPFlow
+        if hasattr(f, 'request'):
```

Comment:
You can then define your own TestFlow(Flow) class in the tests (which has no additional attributes) and use that to test all filters.